runtime/src/prioritization_fee.rs

```diff
@@ -0,0 +1,358 @@
+use {
+    crate::transaction_priority_details::GetTransactionPriorityDetails,
+    solana_measure::measure,
+    solana_sdk::{
+        clock::Slot,
+        pubkey::Pubkey,
+        saturating_add_assign,
+        transaction::{SanitizedTransaction, MAX_TX_ACCOUNT_LOCKS},
+    },
+    std::collections::HashMap,
+};
+
+#[derive(Debug, Default)]
+struct PrioritizationFeeMetrics {
+    // Count of writable accounts in slot; including accounts have minimum prioritization fee equal
+    // to blocker's minimum fee, the number of these irrelevant accounts is the difference between
```

Comment:
I would remove everything in this comment except "Count of writable accounts in slot". I think the comment on relevant_writable_accounts_count is very clear and having a long comment here is making it seem more confusing than it.

Reply:
✅